views/spaces/index.erb

```diff
@@ -1,24 +1,55 @@
 <style>
-	h2 {
-		text-align: center;
-		margin-top: 100px;
-	}
-
-	div {
-		text-align: center;
-		margin-top: 50px;
-	}
+
+  h2 {
+    text-align: center;
+    margin-top: 100px;
+  }
+
+  body {
+    text-align: center;
+    margin-top: 20px;
+  }
+
+  form {
+    text-align: center;
+  }
+
+  div {
+    position: fixed;
+    top: 10;
+    right: 50;
+  }
+
```

Comment:
We can move all css stuff to the style.css file and keep the views cleaner. To do so, you'll need to add the following line to the top of each view, between the <head> tags: <link rel="stylesheet" href="/style.css"/>